test/specs/utilities.test.js

```diff
@@ -5,6 +5,42 @@ const expect = chai.expect;
 
 const utilities = require('../../modules/utilities');
 
+const testRoutes = {
+    sections: {
+        purple: {
+            path: '/purple',
+            pages: {
+                monkey: {
+                    path: '/monkey/dishwasher',
+                    live: true,
+                    isWildcard: false,
+                    isPostable: false,
+                    aliases: ['/green/orangutan/fridge']
```

Comment:
💚  🐒 ❄️ 

(this is the best I could do with emoji)

Reply:
Also important question, is this a green fridge for orangutan or a fridge for _green orangutan_?